README.md

```diff
@@ -93,13 +93,17 @@ If you're still using the [legacy ESLint configuration file format](https://esli
 
 ## References
 
-You might consider using other popular type assertion libraries in the TypeScript ecosystem:
+You might consider using other popular type assertion libraries:
```

Comment:
I like the delineation between "library" and "tool" in theory, but in practice I think it falls apart a bit on Vitest. Vitest itself is a tool, but not a type assertion focused one. What it exports is kind of a superset of a type assertion library...

Proposal: how about this, with one single alphabetized list after it?

```suggestion
You might consider using other popular libraries and tools that can run type assertions:
```

(or some phrasing like that)

Reply:
Thanks. I agree, it sounds better like this.